pontoon/sync/tasks.py

```diff
@@ -127,6 +128,15 @@ def sync_project(
         full_scan=force
     )
 
+    new_entities = new_entities.result
```

Comment:
If `new_entities` is a list, it can't have a `result` property.

Reply:
Calling function.delay() returns an EagerResult object which contains the return value of function() in its `result` attribute (which will be a list that we need). That was something new which I learned while working on this. and to differentiate between return types I should have used different variable names and added comments to it :/ 

Either way, we will not be needing it based on your previous comments :)

Reply:
Ah, thanks for clarifying!